CanHazFunny/CanHazFunny/JokeService.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace CanHazFunny
 {
-    public class JokeService
+    public class JokeService : IService
```

Comment:
`Service` itself is a very ambiguous name, hence why we used the name `JokeService`. This would be similar to naming something `Controller`, when really we need the what the class is controlling. 
```suggestion
    public class JokeService : IJokeService
```